.travis.yml

```diff
@@ -35,7 +35,7 @@ script:
   - npm test --silent
   - npm run cov:report
   - npm run test:artifacts
-  - 'if [ "$TRAVIS_TAG" != "master" && "$TRAVIS_PULL_REQUEST" = "false" ]; then npm run test:release; fi'
+  - 'if [[ "$TRAVIS_TAG" != "master" && "$TRAVIS_PULL_REQUEST" = "false" ]]; then npm run test:release; fi'
```

Comment:
Why do we need double brackets?

Reply:
"&&" operator requires double brackets - https://stackoverflow.com/questions/15993062/bash-scripting-missing/15993083. Without using that I was getting an error